[2022-12-10 16:11:28,916] {processor.py:163} INFO - Started process (PID=127) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:11:28,939] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:11:29,022] {logging_mixin.py:109} INFO - [2022-12-10 16:11:29,021] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:11:59,078] {logging_mixin.py:109} INFO - [2022-12-10 16:11:59,077] {timeout.py:36} ERROR - Process timed out, PID: 127
[2022-12-10 16:11:59,233] {logging_mixin.py:109} INFO - [2022-12-10 16:11:59,093] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_taxizone_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_taxizone_dag.py", line 10, in <module>
    from airflow.providers.google.cloud.operators.bigquery import BigQueryCreateExternalTableOperator
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 38, in <module>
    from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook, BigQueryJob
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 33, in <module>
    from google.cloud.bigquery import (
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/__init__.py", line 35, in <module>
    from google.cloud.bigquery.client import Client
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 67, in <module>
    from google.cloud.bigquery._helpers import _del_sub_prop
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 963, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 906, in _find_spec
  File "<frozen importlib._bootstrap_external>", line 1280, in find_spec
  File "<frozen importlib._bootstrap_external>", line 1252, in _get_spec
  File "<frozen importlib._bootstrap_external>", line 1356, in find_spec
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 37, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/data_ingestion_taxizone_dag.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#reducing-dag-complexity, PID: 127
[2022-12-10 16:11:59,673] {processor.py:690} INFO - Deactivated 1 DAGs which are no longer present in /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:11:59,695] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:12:00,002] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 31.283 seconds
[2022-12-10 16:12:31,561] {processor.py:163} INFO - Started process (PID=160) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:12:31,677] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:12:31,679] {logging_mixin.py:109} INFO - [2022-12-10 16:12:31,679] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:13:01,770] {logging_mixin.py:109} INFO - [2022-12-10 16:13:01,769] {timeout.py:36} ERROR - Process timed out, PID: 160
[2022-12-10 16:13:01,945] {logging_mixin.py:109} INFO - [2022-12-10 16:13:01,833] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_taxizone_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_taxizone_dag.py", line 10, in <module>
    from airflow.providers.google.cloud.operators.bigquery import BigQueryCreateExternalTableOperator
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 38, in <module>
    from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook, BigQueryJob
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 33, in <module>
    from google.cloud.bigquery import (
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/__init__.py", line 35, in <module>
    from google.cloud.bigquery.client import Client
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/client.py", line 74, in <module>
    from google.cloud.bigquery import _pandas_helpers
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/_pandas_helpers.py", line 67, in <module>
    from google.cloud.bigquery import schema
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/schema.py", line 21, in <module>
    from google.cloud.bigquery_v2 import types
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery_v2/__init__.py", line 18, in <module>
    from .types.encryption_config import EncryptionConfiguration
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery_v2/types/__init__.py", line 17, in <module>
    from .model import (
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery_v2/types/model.py", line 39, in <module>
    class Model(proto.Message):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 37, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/data_ingestion_taxizone_dag.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#reducing-dag-complexity, PID: 160
[2022-12-10 16:13:02,297] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:13:02,585] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 31.313 seconds
[2022-12-10 16:13:34,295] {processor.py:163} INFO - Started process (PID=193) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:13:34,334] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:13:34,406] {logging_mixin.py:109} INFO - [2022-12-10 16:13:34,406] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:14:04,550] {logging_mixin.py:109} INFO - [2022-12-10 16:14:04,549] {timeout.py:36} ERROR - Process timed out, PID: 193
[2022-12-10 16:14:04,659] {logging_mixin.py:109} INFO - [2022-12-10 16:14:04,577] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_taxizone_dag.py
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'oauth2client'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_taxizone_dag.py", line 10, in <module>
    from airflow.providers.google.cloud.operators.bigquery import BigQueryCreateExternalTableOperator
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/operators/bigquery.py", line 38, in <module>
    from airflow.providers.google.cloud.hooks.bigquery import BigQueryHook, BigQueryJob
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py", line 46, in <module>
    from googleapiclient.discovery import Resource, build
  File "/home/airflow/.local/lib/python3.7/site-packages/googleapiclient/discovery.py", line 60, in <module>
    from googleapiclient import _auth
  File "/home/airflow/.local/lib/python3.7/site-packages/googleapiclient/_auth.py", line 33, in <module>
    import oauth2client
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 37, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/data_ingestion_taxizone_dag.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#reducing-dag-complexity, PID: 193
[2022-12-10 16:14:05,045] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:14:05,333] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 31.152 seconds
[2022-12-10 16:14:36,584] {processor.py:163} INFO - Started process (PID=224) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:14:36,675] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:14:36,712] {logging_mixin.py:109} INFO - [2022-12-10 16:14:36,712] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:15:06,775] {logging_mixin.py:109} INFO - [2022-12-10 16:15:06,775] {timeout.py:36} ERROR - Process timed out, PID: 224
[2022-12-10 16:15:06,869] {logging_mixin.py:109} INFO - [2022-12-10 16:15:06,823] {dagbag.py:334} ERROR - Failed to import: /opt/airflow/dags/data_ingestion_taxizone_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 331, in _load_modules_from_file
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/data_ingestion_taxizone_dag.py", line 12, in <module>
    import pyarrow.parquet as pq
  File "/home/airflow/.local/lib/python3.7/site-packages/pyarrow/parquet.py", line 34, in <module>
    import pyarrow._parquet as _parquet
  File "<frozen importlib._bootstrap>", line 416, in parent
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/timeout.py", line 37, in handle_timeout
    raise AirflowTaskTimeout(self.error_message)
airflow.exceptions.AirflowTaskTimeout: DagBag import timeout for /opt/airflow/dags/data_ingestion_taxizone_dag.py after 30.0s.
Please take a look at these docs to improve your DAG import time:
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#top-level-python-code
* https://airflow.apache.org/docs/apache-airflow/2.2.3/best-practices.html#reducing-dag-complexity, PID: 224
[2022-12-10 16:15:07,237] {processor.py:656} WARNING - No viable dags retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:15:07,513] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 30.983 seconds
[2022-12-10 16:15:38,743] {processor.py:163} INFO - Started process (PID=258) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:15:38,797] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:15:38,842] {logging_mixin.py:109} INFO - [2022-12-10 16:15:38,841] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:16:04,937] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:16:11,542] {logging_mixin.py:109} INFO - [2022-12-10 16:16:11,541] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:16:12,329] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 33.655 seconds
[2022-12-10 16:16:43,762] {processor.py:163} INFO - Started process (PID=291) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:16:43,796] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:16:43,830] {logging_mixin.py:109} INFO - [2022-12-10 16:16:43,829] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:16:57,947] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:16:58,094] {logging_mixin.py:109} INFO - [2022-12-10 16:16:58,088] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:16:58,375] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 14.712 seconds
[2022-12-10 16:17:28,918] {processor.py:163} INFO - Started process (PID=323) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:17:28,926] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:17:28,928] {logging_mixin.py:109} INFO - [2022-12-10 16:17:28,927] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:17:33,720] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:17:33,751] {logging_mixin.py:109} INFO - [2022-12-10 16:17:33,751] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:17:33,829] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 4.928 seconds
[2022-12-10 16:18:04,194] {processor.py:163} INFO - Started process (PID=348) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:18:04,202] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:18:04,212] {logging_mixin.py:109} INFO - [2022-12-10 16:18:04,212] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:18:13,381] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:18:13,430] {logging_mixin.py:109} INFO - [2022-12-10 16:18:13,430] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:18:13,654] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 9.485 seconds
[2022-12-10 16:18:44,621] {processor.py:163} INFO - Started process (PID=374) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:18:44,643] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:18:44,664] {logging_mixin.py:109} INFO - [2022-12-10 16:18:44,663] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:18:56,176] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:18:56,270] {logging_mixin.py:109} INFO - [2022-12-10 16:18:56,269] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:18:56,425] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 11.889 seconds
[2022-12-10 16:19:26,872] {processor.py:163} INFO - Started process (PID=398) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:19:26,900] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:19:26,918] {logging_mixin.py:109} INFO - [2022-12-10 16:19:26,918] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:19:36,605] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:19:36,658] {logging_mixin.py:109} INFO - [2022-12-10 16:19:36,658] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:19:36,787] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 9.958 seconds
[2022-12-10 16:20:07,163] {processor.py:163} INFO - Started process (PID=423) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:20:07,178] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:20:07,180] {logging_mixin.py:109} INFO - [2022-12-10 16:20:07,179] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:20:15,845] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:20:15,930] {logging_mixin.py:109} INFO - [2022-12-10 16:20:15,930] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:20:16,085] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 8.972 seconds
[2022-12-10 16:20:46,478] {processor.py:163} INFO - Started process (PID=448) to work on /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:20:46,483] {processor.py:642} INFO - Processing file /opt/airflow/dags/data_ingestion_taxizone_dag.py for tasks to queue
[2022-12-10 16:20:46,495] {logging_mixin.py:109} INFO - [2022-12-10 16:20:46,494] {dagbag.py:500} INFO - Filling up the DagBag from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:20:50,457] {processor.py:654} INFO - DAG(s) dict_keys(['data_ingestion_taxizone_dag']) retrieved from /opt/airflow/dags/data_ingestion_taxizone_dag.py
[2022-12-10 16:20:50,474] {logging_mixin.py:109} INFO - [2022-12-10 16:20:50,473] {dag.py:2396} INFO - Sync 1 DAGs
[2022-12-10 16:20:50,594] {processor.py:171} INFO - Processing /opt/airflow/dags/data_ingestion_taxizone_dag.py took 4.140 seconds
